### PR TITLE
more datetime formats w/ 1900-01-01 default instead of error

### PR DESCRIPTION
**Pull request by WillForan**
KeyError fixed when date DNE but year is a requested output column.
new `title` method a la `date` used in warning messages

### CODE REVIEW

File: zotero-cli [hunk 1]
```diff
@@ -338,7 +338,11 @@ class ZoteroCLI(object):
                         if f in NOTE_FIELDS:
                             d[f] = c.strip()
             if "year" in afields:
-                d['year'] = ZoteroCLI.date(i['data']['date'], i).year
+                dt = i.get('data', {'date': None}).get('date')
```
Comment (dhondta) [inline zotero-cli:341]: Should be `dt = i.get('data', {}).get('date')` instead...

**Review by dhondta — COMMENTED** [review]
Hi @WillForan !
Thank you very much for your improvements.
I would rewrite line 341, otherwise it's fine.

**Comment by dhondta** [comment]
@WillForan Thank you for your contribution !

File: zotero-cli (hunk 1 above)
Comment (WillForan) [reply zotero-cli:341]: ops! And I was even messing around with the analogous title get. I guess I was just really into typing `date`. thanks for the fix (and the pull)!

**Pull request merged**